src/components/common/Chatbox.js

```diff
@@ -31,23 +30,6 @@ const Chatbox = ({ item }) => {
     MUTATION_KEYS.DELETE_ITEM_CHAT_MESSAGE,
   );
   const { mutate: clearChat } = useMutation(MUTATION_KEYS.CLEAR_ITEM_CHAT);
-  const [windowHeight, setWindowHeight] = useState(window.innerHeight);
-
-  useEffect(
-    () => {
-      const handleResize = () => {
-        setWindowHeight(window.innerHeight);
-      };
-      window.addEventListener('resize', handleResize);
-
-      // cleanup eventListener
-      return () => {
-        window.removeEventListener('resize', handleResize);
-      };
-    },
-    // run on first render only
-    [],
-  );
```

Comment:
i figured out how to use the flex poxes properly. 

Use `display: 'flex'` on the container, then `flex: 1; overflow: 'auto'` on the messagesContainer so that it resized properly and finally `flex: none;` on the inputContainer.

I had tried to do so before but without luck. It is nice that it worked this time.